usr/initiator.h

```diff
@@ -309,6 +304,15 @@ typedef struct iscsi_session {
 	 * is complete
 	 */
 	queue_task_t *notify_qtask;
+ 
+	/* connection reopens during recovery */
+	int reopen_cnt;
+	int reopen_max;
+	iscsi_session_r_stage_e r_stage;
+	uint32_t replacement_timeout;
+  /* field 'reopen_qtask' with variable sized type 'queue_task_t'
+   * must be at the end of a struct or class is a GNU extension */
+  queue_task_t reopen_qtask;
```

Comment:
Also indented incorrectly, and also based on iscsiadm_rsp_t being variable sized, which I don't see.